pkg/interop/native/neo/neo.go

```diff
@@ -15,9 +15,10 @@ import (
 
 // AccountState contains info about a NEO holder.
 type AccountState struct {
-	Balance int
-	Height  int
-	VoteTo  interop.PublicKey
+	Balance        int
+	Height         int
+	VoteTo         interop.PublicKey
+	LastGasPerVote int
```

Comment:
And after that we should update the dependencies of interop packages. We have `neo-go/scripts/update_deps.sh` script for that, you should provide the latest commit hash to it.

Reply:
Does this operation need to be done after merge? I got error with last commit in this pr.

Reply:
It should be done before the merge and after squashing the whole set of commits and rebasing onto master. What's the error you've got?

Reply:
404 commit not found error. I will squash commits and try again to figure it out.

Reply:
Still
`github.com/nspcc-dev/neo-go/pkg/interop@c94722b515074f34cae800b1db18e1c7af7a308f: invalid version: unknown revision c94722b515074f34cae800b1db18e1c7af7a308f`